pagopa/pm/spec.json

```
@@ -1818,6 +1818,55 @@
           }
         ]
       }
+    },
+    "/v2/wallet/{idWallet}/payment-status": {
+      "put": {
+        "summary": "change the ability to pay",
+        "operationId": "changePayOption",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "idWallet",
+            "type": "integer",
+            "required": true,
+            "description": "Numeric ID of the wallet"
+          },
+          {
+            "in": "body",
+            "name": "payRequest",
+            "description": "New Status required",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WalletPaymentstatus"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "New wallet State",
+            "schema": {
+              "$ref": "#/definitions/WalletV2Response"
```

```suggestion
              "$ref": "#/definitions/WalletV2"
```
`WalletV2Response` is defined as:

```
  WalletV2Response:
    type: object
    properties:
      data:
        $ref: '#/definitions/WalletV2'
    title: WalletV2Response
```
Shouldn't the response be directly a `WalletV2`? Is the wrapping with `data` required?

---

I guess no but I'm pretty sure no APIs return it directly
Instead `WalletV2Response` is used in some APIs response
https://github.com/pagopa/io-services-metadata/blob/7581f8156667a736f20ea7d847cb71f7b6688391/pagopa/pm/spec.json#L930

btw we can ask for this change